src/Components/SlateEditor/SlateEditor.js

```diff
@@ -140,11 +269,28 @@ export default function SlateEditor() {
           }
         }}
       >
+        {renderMarkdownListMenu()}
```

Comment:
why should this be a function and not a React Element? Ideally it should be a react Functional Element (React.FC)